### ISSUE TITLE
Data expiry limit is too tight

### ISSUE THREAD

**Issue by maphew** [lazarus-recovery/original_ff#5]
Data expiry of 14 days is much too short. There are plenty of forms that I need to access only once a year but still need to re-use previous entries.

The current limit is in place in order to workaround _"performance problems and javascript timeouts, particularly during version upgrades, and while searching encrypted data"_, which are indeed problems. But lets solve those a different way, or let us knowingly choose which suffering we're willing to put up with.

_cc: @Tormen_

**Comment by sierracircle** [comment]
I agree with this. At the very least it should be 30 days, but in the past I would use forms saved for over a year.
